src/seedu/addressbook/parser/Parser.java

```diff
@@ -61,25 +62,25 @@ public Command parseCommand(String userInput) {
         switch (commandWord) {
 
             case AddCommand.COMMAND_WORD:
-                return prepareAdd(arguments);
```

Comment:
As discussed in the tutorial, after converting the parseCommand() method, you should also convert all the methods used inside parseCommand() to static as well.